tests/test_stack.py

```diff
@@ -63,19 +63,19 @@ def test_stack_tree(hlwm):
   - 
     - Monitor 1 ("monitor2") with tag "tag2"
       - Focus-Layer
-        - Client <windowid> "sleep infinity"
+        - Client <windowid> "bash"
```

Comment:
I get the purpose of the PR, but what's the reason for this part?

Reply:
The change in the hlwm bridge affects the window title, which again affects this test case.

Reply:
Ah, I read this wrong. Yeah, makes perfect sense! :+1: